packages/harpguru-core/src/components/harp-cell-accessible/harp-cell-accessible.tsx

```diff
@@ -26,27 +30,49 @@ export const HarpCellAccessible = (
     isActive,
     displayMode,
     activeExperienceMode,
+    touchState,
     baseStyles,
   } = props
 
+  const isPreToggled = [State.BEGAN, State.END].includes(touchState)
+  const isReallyActive =
```

Comment:
Could all of this be figured out in the the  harp-cell parent along with the hook which produces the tap handler function and useEffect hook that's been proposed below?

This should be the last part of this review to be considered since I'm not so sure about it. I'm worried that it would homogenise the information coming in to harp-cell-accessible too much for it to be useful for other animations, like a long press or a "remote" activation driven by a tap on a different cell of the same type. I expect we could work around that by using an enum type rather than a boolean.

Reply:
Done - very pleased with commit https://github.com/js-jslog/harpguru/pull/36/commits/a59e65d81aef42becee10ff76a31b97e54b0300dwith this work